### ISSUE TITLE
IRCv3.2-style CAP LS is sent to IRCv3.1 clients

### ISSUE THREAD

**Issue by progval** [kaniini/mammon#78]
```
1450913302.128 1 -> S: CAP LS
1450913302.128 S -> 1: :MyLittleServer CAP * LS * :server-time account-notify away-notify userhost-in-names account-tag metadata-notify extended-join echo-message cap-notify
1450913302.129 S -> 1: :MyLittleServer CAP * LS sasl
```

[IRCv3.2 capability negociation](http://ircv3.net/specs/core/capability-negotiation-3.2.html#version-in-cap-ls):

> Servers MUST NOT send messages described by this document if
> the client only supports version 3.1.

**Comment by DanielOaks** [comment]
I believe this should be fixed as of c247b7143486f5c9c741a5b2b939f8ead4987448 , but if this is still happening in the latest version I'll investigate and fix it.

**Comment by progval** [comment]
It works.
